packages/kbn-optimizer/limits.yml

```diff
@@ -67,7 +67,7 @@ pageLoadAssetSize:
   searchprofiler: 67080
   security: 95864
   securityOss: 30806
-  securitySolution: 217673
```

Comment:
Ah, actually, we do still need to discuss this increase. It's ultimately a much smaller increase but something we still want to understand before we ship. Have you looked into the changes in the securitySolution entry bundle?

Reply:
https://github.com/elastic/kibana/pull/103554/files#diff-275a715ad4a0767bb0e15e5af016a4b7329354d7a8b9c304a68e675dc4a776b4R8-R15

https://github.com/elastic/kibana/pull/103554/files#diff-ea33cc971faa8246841cc043dc80397217332cc28808bea25b2016bf2ed2a0f1R63

Diving into this via webpack visualizer (PR on left at 510.5k for the security solution chunk and main on right at 439.3k):

![Screen Shot 2021-06-30 at 12 32 59 PM](https://user-images.githubusercontent.com/6766512/123998441-80458800-d99f-11eb-9ac6-a4f5fe1d03fd.png)

I believe the cause for the increase here is the importing of the `CustomAssetsAccordion` component from Fleet, and some minor increase from the few net new components underneath the `management` directory.

Reply:
I think part of the issue is https://github.com/elastic/kibana/pull/103554/files#diff-43a25a8739877e22f75056718469e0f07143ec663341b85804a801e42e610d09R11, which should be an `await import()` inside the lazy component (which is what actually makes the lazy component useful).

Reply:
Same here https://github.com/elastic/kibana/pull/103554/files#diff-22a4e62b808f40d069f52410d81ac0ea357071f7b88a1dcd493023a7f9880a83R9

Reply:
I updated these imports, reverted `limit.yml`, and re-ran the `node scripts/build_kibana_platform_plugins.js --update-limits` script. None of the bundle limits were flagged as being needed to increase once we made this change around the lazy imports.

Reply:
Nice, thanks for working through it with me 😄